guacamole/src/main/webapp/translations/cz.json

```diff
@@ -462,6 +462,7 @@
         "FIELD_OPTION_SERVER_LAYOUT_SV_SE_QWERTY" : "Švédština (Qwerty)",
         "FIELD_OPTION_SERVER_LAYOUT_DA_DK_QWERTY" : "Dánština (Qwerty)",
         "FIELD_OPTION_SERVER_LAYOUT_TR_TR_QWERTY" : "Turečtina (Qwerty)",
+        "FIELD_OPTION_SERVER_LAYOUT_NB_NO_QWERTY" : "Norština (Qwerty)",
```

Comment:
Same, here - the option should be in the most appropriate place, `DA_DK` notwithstanding.